ui.tests/test-module/specs/dropdown/dropdown.runtime.spec.js

```diff
@@ -223,4 +223,13 @@ describe("Form with Dropdown", () => {
           cy.get(`#${id}`).should('have.class', 'cmp-adaptiveform-dropdown--filled');
       });
   });
+
+    it("should have empty value checked when default option is not configured", () => {
```

Comment:
The changes in this PR pertains to the view layer, while the test case is checking model. Please add test case for the changes done.